catalog/core/catalog-core-commands/src/test/java/org/codice/ddf/commands/catalog/SeedCommandTest.java

```diff
@@ -205,6 +212,19 @@ public void testDoesNotDownloadCachedResource() throws Exception {
         verify(catalogFramework, times(2)).query(any(QueryRequest.class));
     }
 
+    @Test
+    public void testDoesNotSkipResources() throws Exception {
```

Comment:
✏️ Perhaps some JavaDoc comments above this test briefly explaining what is being tested and what the inputs mean.